hs_file_types/models/model_program.py

```diff
@@ -408,6 +408,12 @@ def get_html_form_mp_file_types():
                                                                               "will be shown below when you "
                                                                               "save metadata.")
 
+                                        with dom_tags.div(cls="control-group"):
```

Comment:
I'm not sure about the class on this div
seems like most other `.control-group`s are inside of a `.form-group`
in Bootstrap2 and BS1, control-group had different margins (by 10px I think)

In BS3, control-group became form-group so that's confusing...
https://github.com/l0rdn1kk0n/wicket-bootstrap/issues/239

Anyway, for our purposes, I don't see that there is any css that actually uses the `control-group` class except for 
https://github.com/hydroshare/hydroshare/blob/master/hs_communities/static/css/bootstrap-select-1.6.3.min.css#L6
https://github.com/hydroshare/hydroshare/blob/master/theme/static/css/hydroshare_core.css#L2381
which don't apply to this case.

Anyway, I think we might want to use `.form-group` instead of `control-group` here... maybe try it and make sure the margins dont get strange?

Reply:
`.form-group` seems to work.